### ISSUE TITLE
scala-cli fmt can't find artifacts

### ISSUE THREAD

**Issue by armanbilge** [VirtusLab/scala-cli#1071]
```
$ scala-cli --version
0.1.6

$ scala-cli fmt
[error]  Error fetching artifacts:
https://github.com/alexarchambault/scalafmt-native-image/releases/download/v3.5.8/scalafmt-x86_64-pc-linux.gz: not found: https://github.com/alexarchambault/scalafmt-native-image/releases/download/v3.5.8/scalafmt-x86_64-pc-linux.gz

```

**Comment by lwronski** [comment]
@armanbilge Thanks for reporting, it should work now. 

The problem was that [scalafmt-native-image](https://github.com/scala-cli/scalafmt-native-image) wasn't published for the latest `scalafmt` version `3.5.8`. 

Under the hood, ScalaCLI uses a GraalVM native-image version of `scalafmt` which is built in  `scalafmt-native-image` repository.

**Comment by armanbilge** [comment]
Thanks. Yes, I appreciate this feature quite a bit, since I can install scala-cli much more easily than scalafmt or coursier, via sdkman :)
